src/core/Dropdown/Dropdown/Dropdown.tsx

```diff
@@ -222,18 +221,18 @@ class BaseDropdown extends Component<DropdownProps> {
       | Array<ReactElement<DropdownItemProps>>
       | ReactElement<DropdownItemProps>
       | undefined,
-  ): string | undefined {
+  ): ReactNode | undefined {
```

Comment:
This function could be renamed getSelectedValueNode and selectedValueText => selectedValueNode